docs/data/charts/lines/lines.md

```diff
@@ -52,6 +52,16 @@ By default, they are stacked in the order you defined them, with positive values
 
 For more information, see [stacking docs](/x/react-charts/stacking/).
 
+## Missing data
```

Comment:
Another sentence explaining that shorter `daŧa` will automatically filled with `null` could be nice

Reply:
They are not, it's just that `data[10]` returns `undefined` if `data` has less than 10 items.

Basically it's the reason why I use `==` instead of `===`

Reply:
OK technically they are not filled.
But my point is just that we suppor correctly data of different length, we can see it in the example, but the text does not make it clear.

Reply:
I don't have a good enough understanding of the charts DX to have a good opinion here.

For me `undefined` and `null` don't have the same meaning so that make sense to differentiate them.